setup.py

```diff
@@ -50,7 +50,7 @@
     include_package_data=True,
     keywords="aws-msk-iam-sasl-signer-python",
     name="aws-msk-iam-sasl-signer-python",
-    packages=find_packages(),
+    packages=find_packages(exclude=['*tests*']),
```

Comment:
Was there a reason a leading wildcard was added to this? That's going to match things like `attests` or other super-strings. If there's not a specific use case, we should be as targeted as possible about what we're excluding.

```suggestion
    packages=find_packages(exclude=['tests*']),
```

Reply:
@sinnwise Thank you for raising the PR. Once you address the comment from @nateprewitt it should be good to go.